### PR TITLE
[Service/Model] define new API

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Define new function and info handle for machine learning models.

### CODE REVIEW

File: c/include/ml-api-service.h [hunk 1]
```diff
@@ -224,9 +224,83 @@ int ml_service_query_create (ml_option_h option, ml_service_h *handle);
...
+ * @remarks Only one model will be activated with given @a key. If same key is already registered in machine learning service, this returns no error and old model will be deactivated when the flag @a activate is true.
+ * @param[in] key The unique key to indicate the model.
+ * @param[in] path The path to neural network model.
+ * @param[in] activate The flag to set the model to be activated.
+ * @param[in] description Nullable, description for neural network model.
+ * @param[out] version The version of registered model.
+ * @return 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
```
Comment (songgot) [inline c/include/ml-api-service.h:238]: I wonder if ML_ERROR_OUT_OF_MEMORY is necessary.

Comment (jaeyun-jung) [reply c/include/ml-api-service.h:238]: IMO, unnecessary. This does not return new handle and passes args via dbus. Return value would be a json or  string and it does not need large memory.

Comment (songgot) [reply c/include/ml-api-service.h:238]: Thanks for the answer.

**Review by songgot — APPROVED** [review]
LGTM

**Comment by jaeyun-jung** [comment]
I added new function `ml_service_model_get()`.

**Comment by songgot** [comment]
LGTM

**Comment by songgot** [comment]
LGTM

**Review by anyj0527 — APPROVED** [review]
LGTM 👍

File: c/include/ml-api-common.h [hunk 2]
```diff
@@ -462,7 +462,37 @@ int ml_option_destroy (ml_option_h option);
...
+
+/**
+ * @brief A handle of machine learning information, key-value pair.
+ * @since_tizen 7.5
+ */
+typedef void *ml_info_h;
+
+/**
+ * @brief Gets a value of key in ml-info instance.
+ * @since_tizen 7.5
```
Comment (anyj0527) [inline c/include/ml-api-common.h:475]: ` * @remarks The @a value should be released using free().`
This line should be added.

File: c/include/ml-api-service.h [hunk 3]
```diff
@@ -224,9 +224,99 @@ int ml_service_query_create (ml_option_h option, ml_service_h *handle);
...
+ */
+int ml_service_model_activate (const char *name, const unsigned int version);
+
+/**
+ * @brief Gets the information of neural network model with given @a name and @ version.
+ * @since_tizen 7.5
```
Comment (anyj0527) [inline c/include/ml-api-service.h:269]: `* @remarks The @a info should be released using ml_info_destroy().`
remarks tag should be added here and other get_* apis

Comment (jaeyun-jung) [reply c/include/ml-api-service.h:269]: I updated this, plz review the changes again.

**Pull request merged**